tasking/pulp/celery_instance.py

```diff
@@ -3,22 +3,21 @@
 
 from celery import Celery
 
-from pulp.app import settings
-
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", 'pulp.app.settings')
 
+from django.conf import settings
```

Comment:
I believe this is already fixed on 3.0-dev

https://github.com/pulp/pulp/blob/1a5e29cd797ae9300060b36ec234f9f364cfe9e8/tasking/pulp/tasking/celery_instance.py#L6